src/components/footer/index.tsx

```diff
@@ -4,7 +4,7 @@ import { useDispatch } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 
 import { i18n, locales } from '@i18n';
-import { AuthActionType } from '@store/auth';
+import { AuthActionType } from '@store/enums';
```

Comment:
Shouldn\t this be exported from the `auth` branch?

Reply:
No, all enums and interfaces from the store branches have single point for import from `@store`